sklearn/tests/test_common.py

```diff
@@ -367,14 +367,13 @@ def _estimators_that_predict_in_fit():
 def test_pandas_column_name_consistency(estimator):
     _set_checking_parameters(estimator)
     with ignore_warnings(category=(FutureWarning)):
-        with pytest.warns(None) as record:
+        with warnings.catch_warnings(record=True) as record:
             check_dataframe_column_names_consistency(
                 estimator.__class__.__name__, estimator
             )
         for warning in record:
             assert "was fitted without feature names" not in str(warning.message)
-
-
```

Comment:
You have linting issue. 2 blank lines are needed between 2 test functions. You need to revert this change.

Reply:
I apologize for the error; I must have overlooked it. I have fixed the linting error now. Thank you!